access-common/src/main/java/edu/unc/lib/dl/ui/service/PermissionsHelper.java

```diff
@@ -79,26 +79,27 @@ public boolean hasThumbnailAccess(AccessGroupSet principals, BriefObjectMetadata
      * @return
      */
     public boolean hasImagePreviewAccess(AccessGroupSet principals, BriefObjectMetadata metadata) {
-        return hasDatastreamAccess(principals, JPEG_2000, metadata);
+        return hasDatastreamAccess(principals, JP2_ACCESS_COPY, metadata);
     }
 
     /**
      * Returns true if the provided principals have rights to access the
      * requested datastream, and the datastream is present.
      *
      * @param principals agent principals
-     * @param datastream name of datastream being requested
+     * @param datastream datastream being requested
```

Comment:
maybe "type of datastream being requested" instead?

Reply:
fixed